scripts/generate-releases.sh

```diff
@@ -2,12 +2,13 @@
 
 set -euo pipefail
 
-if [[ "$#" -lt 2 || "$#" -gt 3 ]]; then
-    echo "USAGE: ./generate-releases.sh <ENVIRONMENT> <RELEASE_NAME_SUFFIX> [<OPERATOR_INDEX_COMMIT>]"
+if [[ "$#" -lt 2 || "$#" -gt 4 ]]; then
+    echo "USAGE: ./generate-releases.sh <ENVIRONMENT> <RELEASE_NAME_SUFFIX> [<OPERATOR_INDEX_COMMIT>] [<OPERATOR_INDEX_BRANCH>]"
```

Comment:
[nits]
1. In my experience, optional parameters go as `[name]` and required ones go as `<name>`. There's usually no such thing as `[<name>]`.
2. Maybe we don't need `OPERATOR_INDEX_` in the names and rather communicate this in the usage help?

```suggestion
    echo "USAGE: ./generate-releases.sh <ENVIRONMENT> <RELEASE_NAME_SUFFIX> [COMMIT] [BRANCH]"
```